rust/xaynet-core/src/mask/masking.rs

```diff
@@ -482,19 +483,20 @@ mod tests {
                     let mut prng = ChaCha20Rng::from_seed(MaskSeed::generate().as_array());
```

Comment:
I think it would be nice for all the tests that use randomness to:

1. print out the seed they are using
2. have a way to manually set the seed for debugging

That would make test failures reproducible, which is always a challenge when tests use random inputs.

Reply:
nice suggestions, I agree it would be nice to have this capability for reproducibility reasons. we can think about tackling this in a separate issue.